BenchmarkTests/Benchmarks/Package.swift

```diff
@@ -0,0 +1,57 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+import Foundation
+
+let package = Package(
+    name: "DatadogBenchmarks",
+    products: [
+        .library(
+            name: "DatadogBenchmarks",
+            targets: ["Benchmarks"]
+        )
+    ]
+)
+
+func addOpenTelemetryDependency(_ version: Version) {
+    // The project must be open with the 'OTEL_SWIFT' env variable.
+    // Please run 'make benchmark-tests-open' from the root directory.
+    //
+    // Note: Carthage will still try to resolve dependencies of Xcode projects in
+    // sub directories, in this case the project will depend on the default
+    // 'DataDog/opentelemetry-swift-packages' depedency.
```

Comment:
Yes, it is the same root cause: carthage resolving dependencies in all projects.
But it's a different setup: Smoke Tests succeed with automatic package resolution, but Benchmark Tests don't because of #1989